### DISCUSSION TITLE
Welcome to asgs Discussions!

### DISCUSSION THREAD

**Discussion by wwlwpd** [StormSurgeLive/asgs#825] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by notstarboard** [comment]
I'm Josh Port, a grad student at URI GSO working on a real-time, high-resolution storm surge forecasting system for Rhode Island using ASGS. I've been working with Brett and Jason via Slack, but I like the idea of Discussions and wanted to get the ball rolling here. I'll also plug this Discussions page in the ASGS install guide I'm working on. Thanks for all of the help @wwlwpd and @jasonfleming !
